test/distributed/elastic/multiprocessing/redirects_test.py

```diff
@@ -138,3 +138,7 @@ def c_print(i):
             libc.printf(bytes(f"c:{i}\n", "utf-8"))
 
         self._redirect_large_buffer(c_print)
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
I think you also need to either change the test name to be test_redirects instead of redirects_test or add it to the list of additional tests, see https://github.com/pytorch/pytorch/issues/124317